src/Etsy/RequestValidator.php

```diff
@@ -98,18 +99,14 @@ public static function validateData($args, $methodInfo)
 								$arg = @$arg[0];
 							} else {
 								$item_type = self::transformValueType(@gettype($arg[0]));
+								if($item_type == 'string' && preg_match("/^[\s0-9,]+$/", $arg[0])) {  //is comma separated integer string
```

Comment:
The old format was, strangely, causing OAuth to fail with php segmentation faults on multiple machines. I know this is not a problem with this code, but this is useful workaround and 100% works with Etsy API.